### DISCUSSION TITLE
Android-FaceDetection: Make it work without adjusting scaleType

### DISCUSSION THREAD

**Discussion by Mikkelet** [google-ai-edge/mediapipe-samples#422] [Feature Requests]
I'm currently trying to implement FaceDetection in my android app using Compose, but the sample project uses views, so Im trying to adapt. In the sample project, the camera has its scaleType set to FIT_START. I thought it was odd, but without it, the overlay 
does not fit. Since my project is using compose, scaleType does not have any effect, and my overlay is offset.

I also checked the face landmark library sample project, where the preview scaleType is set to FILL_START. So both sample projects are using *different* scaletypes? 

I don't see how this is the intended implementation, but any suggestions on how to fix this is welcome!!

**Comment by PaulTR** [comment]
Hey. Check out object detection - that one also has a compose
implementation that might be helpful. I have plans to update all of them at
some point when I have time, but ¯\_(ツ)_/¯

**Reply by Mikkelet** [reply]
Ah that sounds great! I dont see the impl on main, is it in another branch?

**Comment by PaulTR** [comment]
https://github.com/google-ai-edge/mediapipe-samples/tree/main/examples/object_detection/android-jetpack-compose
